packages/launch/package.json

```diff
@@ -26,7 +26,7 @@
 	"types": "dist/types",
 	"scripts": {
 		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && npx rimraf dist",
-		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json && copy-assets && npx copyfiles ./styles/** dist",
```

Comment:
I got an error because the `./styles/**` shell expansion, and when looking into it I found there are no `./styles/**` at all in this package, so I removed it.

Reply:
cool!